wizard/src/utils/file_generation.ts

```diff
@@ -36,16 +37,33 @@ export function generateSvMCF(csv: CsvData, mappings: Mapping): string {
   return "";
 }
 
+// convert Mapping object to a regular javascript object so it can be converted
+// into a JSON string
+function mappingToObject(mapping: Mapping): Record<string, MappingVal> {
+  if (_.isEmpty(mapping)) {
```

Comment:
is this really needed? If mapping is empty, the for loop won't be hit, right? If so, then the empty check isn't needed?

Reply:
ah yes, removed